src/Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// Implementation for the Send-MailMessage command.
     /// </summary>
+    [Obsolete("This cmdlet only supports sending mail from insecure SMTP servers. While there is no immediate replacement available in PowerShell, we recommend you do not use Send-MailMessage at this time.")]
```

Comment:
"from"? Maybe "insecure connect to"?

Reply:
I pinged @joeyaiello 	who wrote the message